backend/src/main/java/heartbeat/controller/report/dto/response/DeploymentFrequencyOfPipeline.java

```diff
@@ -0,0 +1,36 @@
+package heartbeat.controller.report.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.text.DecimalFormat;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class DeploymentFrequencyOfPipeline {
+
+	private String name;
+
+	private String step;
+
+	private String deploymentFrequency;
+
+	private List<DailyDeploymentCount> dailyDeploymentCounts;
+
+	public DeploymentFrequencyOfPipeline(String name, String step, List<DailyDeploymentCount> dailyDeploymentCounts) {
+		this.name = name;
+		this.step = step;
+		this.dailyDeploymentCounts = dailyDeploymentCounts;
+	}
+
+	public void setDeploymentFrequency(Double deploymentFrequency) {
+		DecimalFormat df = new DecimalFormat("0.00");
```

Comment:
"0.00" there is another variable in `backend/src/main/java/heartbeat/controller/report/dto/response/AvgDeploymentFrequency.java`

Reply:
Define '0.00' as a common constant in a class called Constants.